computation_graph/run.py

```diff
@@ -479,7 +479,7 @@ def to_callable_with_side_effect(
     )
 
 
-to_callable = gamla.curry(to_callable_with_side_effect)(gamla.just(gamla.just(None)))
+to_callable = gamla.curry(to_callable_with_side_effect)(debug.mermaid_computation_trace)
```

Comment:
are we sure we want this enabled in prod always ?

Reply:
removed by default